.github/setup-java.sh

```diff
@@ -13,12 +13,16 @@ env | grep -i openjdk
 ./.github/setup-vsmartcard.sh
 
 # Javacard SDKs
-git clone https://github.com/martinpaljak/oracle_javacard_sdks.git
+if [ ! -d "oracle_javacard_sdks" ]; then
+	git clone https://github.com/martinpaljak/oracle_javacard_sdks.git
+fi
 export JC_HOME=$PWD/oracle_javacard_sdks/jc222_kit
 export JC_CLASSIC_HOME=$PWD/oracle_javacard_sdks/jc305u3_kit
 
 # jCardSim
-git clone https://github.com/arekinath/jcardsim.git
+if [ ! -d "jcardsim" ]; then
+	git clone https://github.com/Jakuje/jcardsim.git
```

Comment:
The bug you've found should be reported back.

Reply:
It is reported to the parent project: https://github.com/licel/jcardsim/pull/174 -- I hope if it will be accepted, it can be pulled by @arekinath later. But I can report it also there if there will be no follow-up for some time.